bundles/kernel-gce

```diff
@@ -2,7 +2,7 @@
 # [DESCRIPTION]: Run the kvm specific kernel
```

Comment:
@bryteise. I requested a revision to this description via email because original description is a duplicate of `kernel-kvm.`